### ISSUE TITLE
How can I use this with streaming media?

### ISSUE THREAD

**Issue by DysaniazzZ** [bravobit/FFmpeg-Android#13]
Hi there, I'm new to FFmpeg. I just want to use `ffprobe` to get some info from streaming media on android. I write my codes on this sample module and run it on my device, but I get an error log which says `Protocol not found`. Can you please help me with this? Thx a lot.

BTW, my command array is this: 

    {"-v", "error", "-show_entries", "stream=codec_name,height,width:format=filename,format_name", "-of", "json", "http://223.75.11.92:9090/live/cctv1hd_1500.m3u8"}

And I do it well on my Mac's terminal, which is:

    ffprobe -v error -show_entries stream=codec_name,height,width:format=filename,format_name -of json http://223.75.11.92:9090/live/cctv1hd_1500.m3u8

And the snapshot of the error log:
<img width="1136" alt="screen shot 2018-01-18 at 20 28 27" src="https://user-images.githubusercontent.com/19492924/35098948-1408cfba-fc91-11e7-9b39-739190c022b1.png">
<img width="1155" alt="screen shot 2018-01-18 at 20 29 57" src="https://user-images.githubusercontent.com/19492924/35098956-18e39c36-fc91-11e7-9351-5a53d346b163.png">

**Comment by Brianvdb** [comment]
Network capabilities are currently disabled in the binaries.

**Comment by Brianvdb** [comment]
This issue has been fixed in version 1.1.1. I tested your command and it works fine with that version. Make sure you have internet permission defined in your AndroidManifest.

```gradle
dependencies {
    implementation 'nl.bravobit:android-ffmpeg:1.1.1'
}
```

**Comment by DysaniazzZ** [comment]
Excellent job, my bro. You saved my day!
